lib/services/nodes-api-service.js

```diff
@@ -509,6 +509,26 @@ function nodeApiServiceFactory(
     };
 
     /**
+     * Remove the tag from a list of nodes
+     * @param  {String}     tagName
+     * @return {Promise}
+     */
+    NodeApiService.prototype.masterDelTagById = function(tagName) {
+        return Promise.resolve().then(function() {
+            assert.string(tagName, 'tag must be a string');
+        })
+        .then(function() {
+            return waterline.nodes.findByTag(tagName);
+        })
+        .map(function(node) {
+            return waterline.nodes.remTags(node.id, tagName)
+                .then(function()
+                {return node.id; });
```

Comment:
A style suggestion, Could you please rewrite this code though it has only one line, just like what you did in line#543.